tests/acceptance/docker/src/run-tests.sh

```diff
@@ -7,27 +7,7 @@ git config --global advice.detachedHead false
 
 ## CONFIGURE TEST
 
-if [ "$TEST_SOURCE" = "oc10" ]
-then
```

Comment:
now I am thinking, we might need this if statement because of https://github.com/owncloud/ocis/pull/9914/files#diff-8df1b9512027dc420bf56d9917f282e322367d4b57ece490b6526b9eebadd505

Reply:
if that's the case then we can keep the changes from: https://github.com/owncloud/ocis/pull/9914/files#diff-8df1b9512027dc420bf56d9917f282e322367d4b57ece490b6526b9eebadd505